relay-quotas/src/redis.rs

```diff
@@ -797,4 +835,58 @@ mod tests {
             vec![false]
         );
     }
+
+    fn redis_quota_for_metric_buckets(window: u64, limit: u64) -> RedisQuota<'static> {
+        let quota = Quota {
+            id: Some("foo".to_owned()),
+            categories: DataCategories::new(),
+            scope: QuotaScope::Global,
+            scope_id: None,
+            window: Some(window),
+            limit: Some(limit),
+            reason_code: None,
+        };
+
+        let inner_scoping = Scoping {
+            organization_id: 69420,
+            project_id: ProjectId::new(42),
+            project_key: ProjectKey::parse("a94ae32be2584e0bbd7a4cbb95971fee").unwrap(),
+            key_id: Some(4711),
+        };
+
+        let static_inner_scoping: &'static Scoping = Box::leak(Box::new(inner_scoping));
```

Comment:
Still not a fan of the `Box::leak`, just splitting this setup function in 2 gets rid of the lifetime issues:

```
let quota = build_quota();
let quota = build_redis_quota(&quota);
```